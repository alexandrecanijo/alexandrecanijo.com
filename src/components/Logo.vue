<template>
  <div class="logo">
    <h1>alexandre<span>canijo</span>.com</h1>
    <h2>web developer @ work</h2>
  </div>
</template>

<script>
export default {
  name: 'AcLogo',
};
</script>

<style lang="scss">
.logo {
  h1 {
    font-size: 1.9rem;
    line-height: 3rem;
    font-weight: bold;

    span {
      color: #c75000;
    }
  }

  h2 {
    font-size: 1rem;
    line-height: 2rem;
    font-weight: bold;
  }
}

@media (min-width: $tablet) {
  .logo {
    h1 {
      font-size: 4rem;
      line-height: 6rem;
    }

    h2 {
      font-size: 2rem;
      line-height: 3rem;
    }
  }
}
</style>
