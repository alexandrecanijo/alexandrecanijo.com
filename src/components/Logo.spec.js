import { shallowMount } from '@vue/test-utils';
import Logo from './Logo.vue';

describe('Logo', () => {
  it('should render the component', () => {
    const wrapper = shallowMount(Logo, {});
    expect(wrapper.html()).toMatchSnapshot();
  });
});
