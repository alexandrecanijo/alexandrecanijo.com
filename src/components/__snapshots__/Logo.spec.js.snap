// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render the component 1`] = `
"<div class="logo">
  <h1>alexandre<span>canijo</span>.com</h1>
  <h2>web developer @ work</h2>
</div>"
`;
